dotnet/samples/Concepts/Agents/ChatCompletion_FunctionTermination.cs

```diff
@@ -22,7 +23,7 @@ public async Task UseAutoFunctionInvocationFilterWithAgentInvocationAsync()
             {
                 Instructions = "Answer questions about the menu.",
                 Kernel = CreateKernelWithChatCompletion(),
-                ExecutionSettings = new OpenAIPromptExecutionSettings() { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions },
+                ExecutionSettings = new AzureOpenAIPromptExecutionSettings() { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions },
```

Comment:
You have the option to leave this code as is and then it will work with both OpenAI or AzureOpenAI

Reply:
Correct, I did this as a work-around until Sergey was able to address a serialization bug that was blocking execution.  I'll revert.